src/AppBundle/Form/AdherentCommitteesType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class AdherentCommitteesType extends AbstractType
+{
+    private $committeesChoices;
+
+    public function __construct(EntityManager $em)
+    {
+        $committees = $em->getRepository(Committee::class)->findBy([], null, 3);
```

Comment:
It's better to use a real and more explicit method from your repository class. Below are some examples of good namings.

```
findNearestCommittees(Coordinates $point, $radius, $max = 3);
findMostRecentCommittees($max);
findMostPopularCommittees($max);
```